seqr/management/commands/lift_project_to_hg38_tests.py

```diff
@@ -0,0 +1,269 @@
+            variant_id = "21-3343353-GAGA-G",
+        )
+        saved_variant.saved_variant_json['genomeVersion'] = GENOME_VERSION_GRCh38
+        saved_variant.save()
+
+        # Test discontinue on a lifted variant
+        mock_input.return_value = 'n'
+        with self.assertRaises(CommandError) as ce:
+            call_command('lift_project_to_hg38', u'--project={}'.format(PROJECT_NAME),
+                    '--es-index={}'.format(ELASTICSEARCH_INDEX))
+
+        self.assertEqual(ce.exception.message, 'Error: found 1 saved variants already on Hg38')
+
+        calls = [
+            mock.call(u'Updating project genome version for {}'.format(PROJECT_NAME)),
+            mock.call('Validating es index test_index'),
+        ]
+        mock_logger.info.assert_has_calls(calls)
+
+        mock_get_es_samples.assert_called_with(ELASTICSEARCH_INDEX)
+
+        # Test discontinue on a failed lift
+        mock_liftover_to_38 = mock_liftover.return_value
+        mock_liftover_to_38.convert_coordinate.side_effect = mock_convert_coordinate
+        mock_logger.reset_mock()
+        mock_input.side_effect = ['y', 'n']
+        with self.assertRaises(CommandError) as ce:
+            call_command('lift_project_to_hg38', u'--project={}'.format(PROJECT_NAME),
+                    '--es-index={}'.format(ELASTICSEARCH_INDEX))
+
+        self.assertEqual(ce.exception.message, 'Error: unable to lift over 1 variants')
+
+        calls = [
+            mock.call(u'Updating project genome version for {}'.format(PROJECT_NAME)),
+            mock.call('Validating es index test_index'),
+            mock.call('Lifting over 3 variants (skipping 1 that are already lifted)')
+        ]
+        mock_logger.info.assert_has_calls(calls)
+
+        calls = [
+            mock.call("chr1", 1562437),
+            mock.call("chr1", 46859832),
+            mock.call("chr1", 248367227),
+        ]
+        mock_liftover_to_38.convert_coordinate.assert_has_calls(calls)
+
+        # Test discontinue on failure of finding a variant in the index
+        mock_get_es_variants.return_value = [ES_VARIANT1]
+        mock_logger.reset_mock()
+        mock_input.side_effect = ['y', 'y', 'n']
+        with self.assertRaises(CommandError) as ce:
+            call_command('lift_project_to_hg38', u'--project={}'.format(PROJECT_NAME),
+                    '--es-index={}'.format(ELASTICSEARCH_INDEX))
+
+        self.assertEqual(ce.exception.message, 'Error: unable to find 1 lifted-over variants')
+
+        calls = [
+            mock.call(u'Updating project genome version for {}'.format(PROJECT_NAME)),
+            mock.call('Validating es index test_index'),
+            mock.call('Lifting over 3 variants (skipping 1 that are already lifted)')
+        ]
+        mock_logger.info.assert_has_calls(calls)
+
+        project = Project.objects.get(name=PROJECT_NAME)
+        expected_families = {sv.family for sv in SavedVariant.objects.filter(family__project=project)}
```

Comment:
Please do this as a family query filtering on a hardcoded set of guids. Test data is not dynamic, it should fail when things change. You should always hard code what you are expecting

Reply:
`expected_families` is a set of SavedVariant model objects which normally are returned from queries. I think it is hard to provide statical hard coded data for these objects. Can I use `mock.ANY` to replace this expected called parameter?